app/pandas.ipynb

```diff
@@ -208,18 +215,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Average age\"] = df[[\"Age\"]].mean(axis=1)",
```

Comment:
The calculation of `Average age` seems redundant since it calculates the mean of a single column, which will always be the value itself. Consider revising this logic to calculate the average age across all entries, if that is the intended task.